components/Header/index.js

```diff
@@ -9,4 +9,37 @@
 //  </div >
 // And add it to the DOM in the .headerContainer component
 
-function Header() {}
+function Header() {
+
+    const divCreator = document.createElement('div');
+    const h1Creator = document.createElement('h1');
+    const spanCreator = document.createElement('span');
+    const dateCreator = document.createElement('span');
+
+    divCreator.classList.add('header');
+    spanCreator.classList.add('temp');
+    dateCreator.classList.add('date');
+
+    h1Creator.textContent = 'Lambda Times';
+    spanCreator.textContent = '98';
+    dateCreator.textContent = 'SMARCH 28, 2019';
+
+    const containerOne = document.getElementsByName('.headerContainer');
```

Comment:
I think the issue was that the classname was changed so the class name is `.header-container`